packages/react-widgets/src/widgets/utils/sortLayersById.js

```diff
@@ -0,0 +1,16 @@
+import { assert } from '@carto/react-core';
+
+/**
+ * Sorts a list of layers based on a list of id property values
+ * @param {any[]} layers list of layers to sort
+ * @param {string[]} layerOrder values to sort by
+ * @returns sorted array
+ */
+export default function sortLayersById(layers = [], layerOrder = []) {
+  const uniqueValues = [...new Set(layerOrder)];
```

Comment:
Remove this, it's something up to the user and even if there are repeated values it doesn't affect us.